templates/gson/Project.cshtml

```diff
@@ -0,0 +1,66 @@
+<Project Sdk="Xamarin.Legacy.Sdk">
+  <PropertyGroup>
+    <TargetFramework>MonoAndroid9.0</TargetFramework>
```

Comment:
maybe:

```
    <TargetFrameworks>MonoAndroid9.0;net6.0-android</TargetFrameworks>
```
??

Reply:
Good callout.  I'd prefer to get these moved over without changes, and then we can worry about multitargeting and releasing updated versions.